Description: make kdebase-runtime compile with attica >= 0.1.4
 No new features added. The patch just fixes FTBFS.
Author: Thibaut Gridel <tgridel@free.fr>
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=583019
Forwarded: not-needed
Last-Update: 2010-05-29

--- a/attica/kdeplugin/kdeplatformdependent.h
+++ b/attica/kdeplugin/kdeplatformdependent.h
@@ -47,6 +47,11 @@ public:
     KdePlatformDependent();
     virtual ~KdePlatformDependent();
     virtual QList<QUrl> getDefaultProviderFiles() const;
+    virtual void addDefaultProviderFile(const QUrl& url){}
+    virtual void removeDefaultProviderFile(const QUrl& url){}
+    virtual void enableProvider(const QUrl& baseUrl, bool enabled) const{}
+    virtual bool isEnabled(const QUrl& baseUrl) const{return true;}
+
     virtual QNetworkReply* post(const QNetworkRequest& request, const QByteArray& data);
     virtual QNetworkReply* post(const QNetworkRequest& request, QIODevice* data);
     virtual QNetworkReply* get(const QNetworkRequest& request);
